src/main/scala/nl.knaw.dans.dccd.convert/Configuration.scala

```diff
@@ -29,7 +29,9 @@ object Configuration {
   def apply(): Configuration = {
     val home = Paths.get(System.getProperty("app.home"))
     val cfgPath = Seq(
-      Paths.get(s"/etc/opt/dans.knaw.nl/dccd-nl.knaw.dans.dccd.nl.knaw.dans.dccd.convert-export/"),
+      //Paths.get(s"/etc/opt/dans.knaw.nl/dccd-convert-export/"),
+      //Paths.get(s"/etc/opt/nl.knaw.dans/dccd-convert-export/"),
+      Paths.get(s"/etc/opt/dans.knaw.nl/dccd-convert-export/"),
```

Comment:
I think the third one is the correct one indeed. However, since you don't substitute anything, the leading `s` is not needed here.